releasenotes/notes/change-job-shop-scheduling-generator-f47e34d5a03deb6e.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
```

Comment:
We should also add a comment under `upgrade notes` about the removal of some decision variables.